tests/phpunit/tests/block-template-utils.php

```diff
@@ -161,6 +161,53 @@ public function test_build_block_template_result_from_file() {
 		$this->assertEmpty( $template_part->modified );
 	}
 
+	/**
+	 * @dataProvider data_build_block_template_result_from_file_injects_theme_attribute
+	 */
+	public function test_build_block_template_result_from_file_injects_theme_attribute( $filename, $expected_content ) {
+		$template = _build_block_template_result_from_file(
+			array(
+				'slug' => 'single',
+				'path' => __DIR__ . '/../data/templates/' . $filename,
```

Comment:
We should be able to use the `DIR_TESTDATA` constant here like so:

```php
DIR_TESTDATA . "/templates/$filename";
```
This makes it safe to move this test file somewhere else in future if needed.

Reply:
3d552d5502